src/ldf-client-ui.js

```diff
@@ -673,9 +711,102 @@ require('yasgui-yasqe/dist/yasqe.css'); // Make webpack import the css as well
       if (this._writeResult) {
         this._resultCount++;
         this._writeResult(result);
+
+        // Adding wkt points on a map
+        for (const value in result) {
+          // This code adds points to the map
+          var checkingResult = `${result[value]}`;
```

Comment:
Can we immediately name this `wktIncludingVariable`? So there's no need for two variables for the same thing.

Reply:
Edited